### PR TITLE
small bug fix for psis_lw

### PR DESCRIPTION
**Pull request by junpenglao**
cherry picked https://github.com/pymc-devs/pymc3/pull/2952/files
also expose psis_lw (it is quite a handy function that can be use in many diagnoistics)

### CODE REVIEW

File: arviz/tests/test_stats.py [hunk 1]
```diff
@@ -104,7 +104,7 @@ def test_waic():
 
         step = pm.Metropolis()
         trace = pm.sample(100, step)
-        calculated_waic = pm.waic(trace)
+        calculated_waic = waic(trace)
```
Comment (aloctavodia) [inline arviz/tests/test_stats.py]: There is no `modelcontext` in ArviZ, thus you should explicitly pass the model to  `waic`

**Comment by aloctavodia** [comment]
Thanks for fixing this (including all the `pm.` I miss!!!)

File: arviz/tests/test_stats.py [hunk 2]
```diff
@@ -117,3 +117,8 @@ def test_waic():
 
     assert_almost_equal(calculated_waic.WAIC, actual_waic, decimal=2)
```
Comment (aloctavodia) [inline arviz/tests/test_stats.py]: This should be lowercase (`calculated_waic.waic`, `calculated_waic.waic_se`). Sorry, this is my fault for copy&pasting from PyMC3.

Comment (junpenglao) [reply arviz/tests/test_stats.py]: LOL, thanks! (I really should have run the tests locally first).

**Pull request merged**